src/main/java/teammates/common/datatransfer/CourseRoster.java

```diff
@@ -86,4 +112,150 @@ private void populateInstructorListByEmail(List<InstructorAttributes> instructor
         }
     }
 
+    /**
+     * Adds giver and recipient of response to emailToNameTable.
+     * @param response Feedback response
+     * @param question Feedback question
+     */
+    public void addEmailNamePairsToTable(FeedbackResponseAttributes response,
+                                          FeedbackQuestionAttributes question) {
+        // keys of emailToNameTable are participantIdentifiers,
+        // which consists of students' email, instructors' email, team names, or %GENERAL%.
+        // participants identifiers of anonymous responses are not anonymised in the tables
+        addEmailNamePairsToTable(response, question, EMAIL_NAME_PAIR);
+    }
+
+    /**
+     * Adds giver and recipient of response to emailToTeamNameTable.
+     * @param response Feedback response
+     * @param question Feedback question
+     */
+    public void addEmailTeamNamePairsToTable(FeedbackResponseAttributes response, FeedbackQuestionAttributes question) {
+        addEmailNamePairsToTable(response, question, EMAIL_TEAMNAME_PAIR);
+    }
+
+    private void addEmailNamePairsToTable(FeedbackResponseAttributes response,
+                                          FeedbackQuestionAttributes question,
+                                          int pairType) {
+        if (question.giverType == FeedbackParticipantType.TEAMS
```

Comment:
I have copy pasted this code from `FeedbackSessionResultsBundle`. If this looks fine then we can also remove xxxxtables form `FeedbackSessionResultsBundle`.

Reply:
I belive you can do that also?